packages/devtools_app/lib/src/flutter/utils.dart

```diff
@@ -18,16 +17,18 @@ Future<void> launchUrl(String url, BuildContext context) async {
   }
 }
 
-/// Attempts to copy a bunch of `lines` to the clipboard.
-Future<void> copyToClipboard(List<String> lines, BuildContext context) async {
+/// Attempts to copy a String of `data` to the clipboard.
+///
+/// Shows a `successMessage` [Notification] on the passed in `context`.
+Future<void> copyToClipboard(
+    String data, String successMessage, BuildContext context) async {
```

Comment:
nit: trailing comma

Reply:
Hahaha thanks for noticing. I tried to catch 'em all, but I'm not good at pokemon :P

Reply:
Done!